lib/preserve_data/save_books.rb

```diff
@@ -0,0 +1,5 @@
+def save_books
+  file_path = "#{DATA_PATH}/books.json"
+  json_data = @books.map { |book| { 'title' => book.title, 'author' => book.author } }.to_json
+  File.write(file_path, json_data)
+end
```

Comment:
- [ ] One of the requirements for this project is to ensure that your app works properly when one or more of the JSON files used to store data are missing. Kindly implement this change so that your app doesn't crash when the files are missing.

#### Hint: Instead of returning, create a new file if the path to the data storage returns false. 👍 😎